script/nnbd_plugins.sh

```diff
@@ -38,13 +38,9 @@ readonly NNBD_PLUGINS_LIST=(
 # building the all plugins app. This list should be kept empty.
 
 readonly NON_NNBD_PLUGINS_LIST=(
-  "camera"
-  "google_maps_flutter" # half migrated
-  # "image_picker"
-  # "in_app_purchase"
-  # "quick_actions"
-  # "sensors"
-  # "wifi_info_flutter"
+  "camera" # partially migrated
```

Comment:
This morning I merged a null safety version of the `camera` plugin. I believe this line can now be removed (it is removed in `master` so maybe a rebase should be sufficient).